drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/models/hud_twenty_twenty_two_to_twenty_twenty_four/current_living_situation/change_living_situation.rb

```diff
@@ -10,6 +10,7 @@ class ChangeLivingSituation
 
     def process(row)
       situation = row['CurrentLivingSituation'].to_i
+      row['LivingSituation2022'] = situation
```

Comment:
@awisema do I need to do more than this?  Like do I need to allow the new columns in some way?